SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Supported Versions
+
+We support fixing security issues on the following releases:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 3.7.x   | :white_check_mark: |
+| 3.6.x   | :white_check_mark: |
+| <= 3.5  | :x:                |
+| 2.10.x  | :white_check_mark: |
+| <= 2.9  | :x:                |
+
+## Reporting a Vulnerability
+
+If you’ve found a security issue in CakePHP, please use the following procedure 
+instead of the normal bug reporting system. Instead of using the bug tracker, 
+mailing list or IRC please send an email to security [at] cakephp.org. Emails 
```

Comment:
We no longer have a mailing list, so perhaps mention Slack and Discourse forum instead?